app/views/questions/received_questions.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container received-questions">
+  <h1 class="title">Received Questions</h1>
+
+  <% current_user.unanswered_questions.each do |question| %>
+    <div class="list-received-questions">
+      <div class="row">
+        <div class="col">
+          <small class="text-muted">
+            <em>Sent by&nbsp;</em>
+            <% unless question.anonymous %>
```

Comment:
Eu trocaria isso por um if, pra ficar melhor de ler. Aí mudaria os blocos de ordem ou criaria um `question.known_sender?`

Reply:
Nunca use unless unless seja a única alternativa, mas em quase todos os casos o unless nunca é a única alternativa. Às vezes até uma negação no início fica mais legível